torch/onnx/_internal/io_adapter.py

```diff
@@ -487,3 +540,44 @@ def apply(self, model_outputs: Any) -> Sequence[Any]:
                 error_message="Model outputs incompatible with the format that was exported. ",
             )
         return flattened_outputs
+
+
+class PrependParamsAndBuffersAotAutogradInputStep:
+    """Prepend model parameters and buffers to the user input.
+
+    :func:`torch.export.export` lifts model parameters and buffers as model input, thus, they
+    must be added to the user input before the model  is executed.
+
+    """
+
+    def apply(
+        self,
+        model_args: Sequence[Any],
+        model_kwargs: Mapping[str, Any],
+        model: Optional[
```

Comment:
pass model in `__init__` to keep `apply` signature unchanged?

Reply:
I was wondering the same. I think this is a good idea.